tests/src/Integration/Action/EntityPathAliasCreateTest.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\EntityPathAliasCreateTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Core\Entity\ContentEntityType;
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\Action\EntityPathAliasCreate
+ * @group rules_actions
+ */
+class EntityPathAliasCreateTest extends RulesEntityIntegrationTestBase {
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Core\RulesActionInterface
+   */
+  protected $action;
+
+  /**
+   * The mocked alias storage service.
+   *
+   * @var \PHPUnit_Framework_MockObject_MockObject|\Drupal\Core\Path\AliasStorageInterface
+   */
+  protected $aliasStorage;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    // Prepare a content entity type instance.
+    $this->entityType = new ContentEntityType([
+      'id' => 'test',
+      'label' => 'Test',
+      'entity_keys' => [
+        'bundle' => 'bundle',
+      ],
+    ]);
+
+    // Prepare mocked entity manager.
+    $this->entityManager = $this->getMockBuilder('Drupal\Core\Entity\EntityManager')
```

Comment:
Why are you mocking this yourself? The RulesEntityIntegrationTestBase already does that for you.